src/cmd/top/summary_pane.c

```diff
@@ -417,6 +429,8 @@ struct summary_pane *summary_pane_create (struct top *top)
     sum->instance_level = get_instance_attr_int (top->h, "instance-level");
     sum->instance_size = get_instance_attr_int (top->h, "size");
     sum->instance_version = flux_attr_get (top->h, "version");
+    if (flux_get_instance_starttime (top->h, &sum->starttime) < 0)
+        fatal (errno, "error fetching broker.starttime attribute");
```

Comment:
Should `flux top` tolerate a missing broker.starttime?
```
$ src/cmd/flux top
flux-top: error fetching broker.starttime attribute: No such file or directory
```

Reply:
OK, that's easy enough.

Reply:
Pushed a fix for that - it will just use the start time of `top` if the instance starttime is unavailable.  That OK?

Reply:
That seems fine. It would be an increasingly rare corner case anyway.